pkg/master/pod_cache.go

```diff
@@ -192,6 +192,7 @@ func (p *PodCache) computePodStatus(pod *api.Pod) (api.PodStatus, error) {
 	} else {
 		newStatus.Info = result.Status.Info
 		newStatus.PodIP = result.Status.PodIP
+		newStatus.Conditions = result.Status.Conditions
```

Comment:
Just move this newStatus.Conditions assignment to the else clause, to be symmetric with the if clause.